bundle/orchestrate/dags/meltano.py

```diff
@@ -32,75 +32,152 @@
+
+    Args:
+        schedules (list): List of Meltano scheduled jobs.
+    """
+    for schedule in schedules:
+        if not schedule.get("job"):
+            logger.info(
+                f"No DAG's created for schedule '{schedule['name']}'. It was passed to job generator but has no job."
+            )
+            continue
+        if not schedule["cron_interval"]:
+            logger.info(
+                f"No DAG created for schedule '{schedule['name']}' because its interval is set to `@once`."
+            )
+            continue
+
+        base_id = f"meltano_{schedule['name']}_{schedule['job']['name']}"
+        common_tags = DEFAULT_TAGS.copy()
+        common_tags.append(f"schedule:{schedule['name']}")
+        common_tags.append(f"job:{schedule['job']['name']}")
+        interval = schedule["cron_interval"]
+
+        for idx, task in enumerate(schedule["job"]["tasks"]):
+            logger.info(
+                f"Considering task '{task}' of schedule '{schedule['name']}': {schedule}"
+            )
+            args = DEFAULT_ARGS.copy()
+            args["start_date"] = datetime.utcnow()
+            dag_id = f"{base_id}_task{idx}"
+            task_tags = common_tags.copy()
+
+            dag = DAG(
+                dag_id,
+                description=f"Meltano run task[{idx}]: '{task}'",
+                tags=task_tags,
+                catchup=False,
+                default_args=args,
+                schedule_interval=interval,
+                max_active_runs=1,
+            )
+
+            if isinstance(task, Iterable) and not isinstance(task, str):
+                run_args = " ".join(task)
+            else:
+                run_args = task
+
+            elt = BashOperator(
+                task_id=f"run_task_{idx}",
+                bash_command=f"cd {PROJECT_ROOT}; {MELTANO_BIN} run {run_args}",
+                dag=dag,
+            )
+
+            globals()[dag_id] = dag
+            logger.info(
+                f"Task DAG created for schedule '{schedule['name']}', task='{run_args}'"
+            )
+
+
+def create_dags():
+    """Create DAGs for Meltano schedules."""
+    list_result = subprocess.run(
+        [MELTANO_BIN, "schedule", "list", "--format=json"],
+        cwd=PROJECT_ROOT,
+        stdout=subprocess.PIPE,
+        universal_newlines=True,
+        check=True,
     )
+    schedule_export = json.loads(list_result.stdout)
+
+    if schedule_export.get("schedules"):
+        logger.info(f"Received meltano v2 style schedule export: {schedule_export}")
+        _meltano_elt_generator(schedule_export["schedules"].get("elt"))
+        _meltano_job_generator(schedule_export["schedules"].get("job"))
+    else:
+        logger.info(f"Received meltano v1 style schedule export: {schedule_export}")
+        _meltano_elt_generator(schedule_export)
 
-    # register the dag
-    globals()[dag_id] = dag
 
-    logger.info(f"DAG created for schedule '{schedule['name']}'")
+create_dags()
```

Comment:
i wrapped everything in create_dags, so that theoretically we can actually add some tests. I was gonna go ahead and add some repo scaffolding and basic tests for us, but considering how short we are on time @aaronsteers do you me to skip adding tests for now ?